Fix: Dependabot is paused (#6406) (#6414)
DataBiosphere/azul: .github/PULL_REQUEST_TEMPLATE/upgrade.md

- [ ] Added PR # reference to merge commit title
- [ ] Collected commit title tags in merge commit title <sub>but excluded any `p` tags</sub>
- [ ] Moved connected issue to *Merged lower* column in ZenHub

> - [ ] Closed related Dependabot PRs with a comment referencing the corresponding commit in this PR <sub>or this PR does not include any Dependabot related commits</sub>

Reviewer: Good choice of placement.

> - [ ] Dependabot PRs have been closed, with each referencing its corresponding commit in the upgrades PR <sub>or this PR does not include any Dependabot related fix attempts</sub>

Reviewer 2: ```suggestion
- [ ] Closed related Dependabot PRs with a comment referencing the corresponding commit in this PR <sub>or this PR does not include any Dependabot related commits</sub>
```
